data_dir/mv_synchronous_updates.yaml

```diff
@@ -0,0 +1,87 @@
+# LWT test: create and update data using LWT.
+### DML ###
+
+# Keyspace Name
+keyspace: mv_synchronous_ks
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS mv_synchronous_ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: mv_synchronous_cf
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE IF NOT EXISTS mv_synchronous_cf (
+        domain int,
+        published_date int,
```

Comment:
if we're already doing this, why not using a "date" data type (and the appropriate filter below).

Reply:
You can also use timestamp for easier filtering...

Reply:
> if we're already doing this, why not using a "date" data type (and the appropriate filter below).

because with int I can manage view creation easily

Reply:
but you cover less things... same thing we already covering in all other places...

Reply:
changed